dotnet/src/Aot/AotCompatibility.TestApp/AotCompatibility.TestApp.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should this be in the `samples\Demos` folder?

Reply:
The test application serves two purposes: uncovering all AOT-related warnings and running E2E scenarios that use the SK AOT APIs to test them. We can certainly consider a third purpose: demonstrating how certain scenarios can be configured in AOT.

Reply:
Done